worlds/musedash/__init__.py

```diff
@@ -271,15 +269,15 @@ def create_regions(self) -> None:
         # Make a region per song/album, then adds 1-2 item locations to them
```

Comment:
```suggestion
        # Adds 2 item locations per song/album to the menu region
```
Update the comment to reflect these changes

Reply:
Should be fixed up. And removed a later one.